sub/sd_ass.c

```diff
@@ -533,6 +533,9 @@ static void configure_ass(struct sd *sd, struct mp_osd_res *dim,
                             | ASS_OVERRIDE_BIT_COLORS
                             | ASS_OVERRIDE_BIT_BORDER
                             | ASS_OVERRIDE_BIT_SELECTIVE_FONT_SCALE;
+#if LIBASS_VERSION >= 0x01703020
+        set_force_flags |= ASS_OVERRIDE_BIT_BLUR;
```

Comment:
This needs interface changes entry.

Reply:
Updated the option documentation a bit to more accurately reflect what the goal of the option is. I want to clarify that the option has always worked this way, just not adequately documented.


https://github.com/mpv-player/mpv/commit/2f1eb49e9f130c728c09a3458ff8449bc571c0c0 wm4 notes here that alignment is not applied with this option, because it could have unintended consequences. And I also removed applying sub-bold/sub-italic from this option here https://github.com/mpv-player/mpv/commit/d02f03bc2b98bd6e0ed84e7627be7de02dce7a9c

I believe both of these changes are in the spirit of what wm4 intended this option to me, which I also agree with because otherwise `force` and `strip` are not very different and there would be little reason for the option to exist in the first place - so I don't think an interface changes entry is necessary, just the documentation of the option. I could add an interface changes entry if you think it's necessary but I wouldn't know what to say there.

Reply:
Added a changelog entry about blur anyway, I doubt the end user cares about this and just wants to see functional changes